backend/ibutsu_server/db/models.py

```diff
@@ -68,6 +68,7 @@ class Artifact(Model, FileMixin):
     result_id = Column(PortableUUID(), ForeignKey("results.id"), nullable=False)
     filename = Column(Text)
     data = Column(mutable_json_type(dbtype=PortableJSON(), nested=True))
+    upload_date = Column(DateTime)
```

Comment:
If you need to make a default value for this, you can do the following:

```suggestion
    upload_date = Column(DateTime, default=datetime.utcnow)
```